src/App.js

```diff
@@ -1,5 +1,38 @@
+import { 
+  getComputerNumber, 
+  getGameResult, 
+  printGameResult 
+} from './utils/game';
+import { 
+  getInputNumber, 
+  getInputContinueNumber 
+} from './utils/user';
+import { MissionUtils } from '@woowacourse/mission-utils';
+import { GAME_MESSAGE } from './constants/gameMessage';
+
 class App {
-  async play() {}
+  async play() {
+    let computerNumber = await getComputerNumber();
+    let userAnswer = 1;
+
+    await MissionUtils.Console.print(GAME_MESSAGE.gameStart);
+
+    while (userAnswer === 1) {
+      const inputNumber = await getInputNumber();
```

Comment:
숫자 야구 게임에서 input으로 받는경우가 사용자 수와 게임을 재시작할것인지 물어보는 2가지가 있기도 하고
컴퓨터 수 이름은 `computerNumber`여서 `userNumber`로 선언하는게 일관성 있다고 생각합니다..!
```suggestion
      const userNumber = await getInputNumber();
```